README.md

```diff
@@ -0,0 +1,196 @@
+ ```
+To utilize all available CPUs for the command, use the option:
+```bash
+--num-procs=$SLURM_CPUS_PER_TASK
+```
+
+### Usage Example
+After you've set up pyflexplot ([Prerequisites and cloning the repository](#prerequisites-and-cloning-the-repository) and [Getting started](#getting-started)),
+you'll need to specify a configuration file and an output directory.
+Create an output directory using:
+```bash
+dest=test_output/
+```
+Note: The directory will be automatically created if it doesn't already exist.
+
+There are several default config files available under ```src/pyflexplot/data/presets/opr```.
+Furthermore, there are already several default config files available in the directory ```src/pyflexplot/data/presets/opr```.
+To run the program for all presets in the PDF graphics format with the default input data, use:
+ ```bash
+preset='opr/*/all_pdf'
+```
+Alternatively, select a specific preset from the table below:
+| Model            | Type                 | Preset                           |
+|------------------|----------------------|----------------------------------|
+| FLEXPART-IFS     | Global output:       | preset=opr/ifs-hres/all_pdf      |
+| FLEXPART-IFS     | Europe output:       | preset=opr/ifs-hres-eu/all_pdf   |
+| FLEXPART-COSMO   | deterministic output:| preset=opr/cosmo-1e-ctrl/all_pdf |
+| FLEXPART-COSMO   | deterministic output:| preset=opr/cosmo-2e-ctrl/all_pdf |
+| FLEXPART-COSMO-1E| ensemble output:     | preset=opr/cosmo-1e/all_pdf      |
+| FLEXPART-COSMO-2E| ensemble output:     | preset=opr/cosmo-2e/all_pdf      |
+
+After selecting a preset, run pyflexplot interactively:
+ ```bash
+pyflexplot --preset "$preset" --merge-pdfs --dest=$dest
+```
+
+## Developer Notes
+
+As this package was created with the SEN Python blueprint, it comes with a stack of development tools, which are described in more detail on the [Website](https://meteoswiss-apn.github.io/mch-python-blueprint/). Here, we give a brief overview on what is implemented.
+
+### Implemented Debugging Features
+pyflexplot offers several debugging options to assist in troubleshooting and
+refining your workflow (see ```pyflexplot -h```).
+Here are some of the key debugging features:
+ ```
+--pdb / --no-pdb                Drop into debugger when an exception is raised.  [default: no-pdb]
+--preset-cat PATTERN            Show the contents of preset setup files
+--only N                        Only create the first N plots based on the given setup.
+--raise / --no-raise            Raise exception in place of user-friendly but uninformative error message.
+--dry-run                       Perform a trial run with no changes made.
+--merge-pdfs-dry                Merge PDF plots even in a dry run.
+```
+
+### Roadmap to your first Contribution
+
+Generally, the source code of your library is located in `src/<library_name>`. The blueprint will generate some example code in `mutable_number.py`, `utils.py` and `cli.py`. `cli.py` thereby serves as an entry
```

Comment:
This part needs revising still pyflexplot does not include `mutable_number.py` anymore

Reply:
Yeah, removed that part.